docs/sources/source/nlp/ner.rst

```diff
@@ -12,115 +12,120 @@ This tutorial explains how to implement named entity recognition (NER) in NeMo.
+Each line of the text.txt file contains text sequences, where words are separated with spaces. The labels.txt file contains corresponding labels for each word in text.txt, the labels are separated with spaces. Each line of the files should follow the format: [WORD] [SPACE] [WORD] [SPACE] [WORD] (for text.txt) and [LABEL] [SPACE] [LABEL] [SPACE] [LABEL] (for labels.txt). There can be columns in between for part-of-speech tags, as shown on the `CoNLL-2003 website`_.
 
 .. _CoNLL-2003 website: https://www.clips.uantwerpen.be/conll2003/ner/
 
-.. _Preprocessed data: https://github.com/kyzhouhzau/BERT-NER/tree/master/data
+
 
 Training
 --------
 
 .. tip::
 
     We recommend you try this out in a Jupyter notebook. It'll make debugging much easier!
+    See examples/nlp/NERWithBERT.ipynb
 
 First, we need to create our neural factory with the supported backend. How you should define it depends on whether you'd like to multi-GPU or mixed-precision training. This tutorial assumes that you're training on one GPU, without mixed precision. If you want to use mixed precision, set ``amp_opt_level`` to ``O1`` or ``O2``.
 
     .. code-block:: python
 
         nf = nemo.core.NeuralModuleFactory(backend=nemo.core.Backend.PyTorch,
-                                           local_rank=args.local_rank,
-                                           optimization_level=args.amp_opt_level,
-                                           log_dir=work_dir,
-                                           create_tb_writer=True,
-                                           files_to_copy=[__file__])
+                                           local_rank=None,
+                                           optimization_level=nemo.core.Optimization.mxprO0,
+                                           create_tb_writer=True)
 
 Next, we'll need to define our tokenizer and our BERT model. There are a couple of different ways you can do this. Keep in mind that NER benefits from casing ("New York City" is easier to identify than "new york city"), so we recommend you use cased models.
 
 If you're using a standard BERT model, you should do it as follows. To see the full list of BERT model names, check out ``nemo_nlp.huggingface.BERT.list_pretrained_models()``
 
     .. code-block:: python
 
-        tokenizer = NemoBertTokenizer(args.pretrained_bert_model)
-        pretrained_bert_model = nemo_nlp.huggingface.BERT(
-            pretrained_model_name=args.pretrained_bert_model)
+        tokenizer = NemoBertTokenizer(pretrained_model="bert-base-cased")
+        bert_model = nemo_nlp.huggingface.BERT(
+            pretrained_model_name="bert-base-cased",
+            factory=neural_factory)
```

Comment:
factory=neural_factory is no longer necessary

Reply:
done